### PR TITLE
HEVC ENC:10bit support

### PR DESCRIPTION
**Pull request by wudping**
v1:
    add support P010 input format
    add HEVC 10bit encoding

Signed-off-by: Pengfei Qu <Pengfei.Qu@intel.com>
Signed-off-by: wudping <dongpingx.wu@intel.com>

### CODE REVIEW

**Comment by wudping** [comment]
for hevc 10-bits encoding, rebase Pengfei's patch, please help to review.

File: tests/encodehelp.h [hunk 1]
```diff
@@ -184,6 +187,11 @@ static bool process_cmdline(int argc, char *argv[])
     }
 #endif
 
+    if (!inputFourcc) {
```
Comment (xuguangxin) [inline tests/encodehelp.h]: Please add support yamitranscode too

**Comment by wudping** [comment]
updated, please help to review.

**Comment by wudping** [comment]
updated again, please help review, thanks advance.

**Comment by xuguangxin** [comment]
merged as efe74276a6c2f2cb23cd3d77d91418dab3829e33, @mypopydev

**Pull request closed without merging**